008-infinispan-getting-started/README.md

```diff
@@ -0,0 +1,16 @@
+##hotrod-client.properties
```

Comment:
I wrote: `This brings in dependency on Docker, make a note about it in README.md in the root of the repository`.

README.md in the root of the repository :) This file can stay as is.

Add something like this into README.md in the root
```md
## Prerequisites
 - JDK 11
 - Docker

```

Reply:
@rsvoboda ok It was clear, but don't know why I added it there :)